scripts/test-release-workflow-config.sh

```
@@ -0,0 +1,26 @@
+  echo "FAIL: $*" >&2
+  exit 1
+}
+
+grep -q "IMAGE_NAMESPACE: mtg-thomas" "$workflow" \
+  || fail "ci.yml must publish fork images under ghcr.io/mtg-thomas"
+
+if grep -q "jackmusick/bifrost-\(api\|client\)" "$workflow"; then
+  fail "ci.yml still publishes Bifrost images under jackmusick"
+fi
+
+grep -q 'ghcr.io/${{ env.IMAGE_NAMESPACE }}/${{ env.API_IMAGE }}' "$workflow" \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)